migrations/2024-02-12-211114_add_vote_display_mode_setting/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TYPE vote_display_mode_enum AS enum (
+    'Full',
+    'ScoreAndUpvotePercentage',
+    'UpvotePercentage',
+    'HideDownvotes',
+    'HideAll'
+);
+
+ALTER TABLE local_user
+    ADD COLUMN vote_display_mode vote_display_mode_enum DEFAULT 'ScoreAndUpvotePercentage' NOT NULL;
```

Comment:
From jerboa:

![Screenshot_2024-02-12-12-52-32-271-edit_com jerboa_1](https://github.com/LemmyNet/lemmy/assets/930215/5f5905ef-762d-4225-9732-3d986f3ef04c)

You can see that there's a lot of redundant and unecessary info in displaying all these three pieces of info, like we do now in lemmy-ui. 

I'd argue that a score and upvote pct (that only probably needs to be shown when < 90% or something), is a preferred default, as that takes it from 3 pieces of info, to 2 or 1. I see a lot of apps doing this anyway.

Reply:
> (that only probably needs to be shown when < 90% or something)

I would leave this as a implementation detail for the clients to decide